pkg/controller/statusmanager/pod_status.go

```diff
@@ -258,7 +258,7 @@ func (status *StatusManager) SetFromPods() {
 	if icConfigMap, err := util.GetInterConnectConfigMap(status.client.ClientFor("").Kubernetes()); err == nil {
 		// When an upgrade from <= 4.13 is ongoing, the IC configmap exists and exhibits ongoing-upgrade=true.
 		// When multizone control-plane and node have been rolled out (end of phase 2), the configmap is deleted.
-		if ongoingUpgrade, ok := icConfigMap.Data["ongoing-upgrade"]; ok && ongoingUpgrade == "true" {
+		if _, ok := icConfigMap.Data["ongoing-upgrade"]; ok {
```

Comment:
@ricky-rav the code no longer agrees with the comment...

Reply:
Ops, thanks!
While I'm at it, I'll add a couple of more fixes. I'll add the WIP tag to the title and will remove it once I've run all the necessary upgrades :)